data_processing/radiology/proxy_table/generate.py

```diff
@@ -232,32 +232,41 @@ def update_graph(config_file):
         'MODALITY',
     ]
 
-    dataset_props = list ( set(dataset_ext_properties).intersection(set(cfg.get_keys(name=DATA_CFG))))
-    dataset_props.insert(0, 'LANDING_PATH')
-    dataset_props.insert(0, 'DATASET_NAME')
-    
+    dataset_props = list ( set(dataset_ext_properties).intersection(set(cfg.get_keys(name=const.DATA_CFG))))
+
     prop_string = ','.join(['''{0}: "{1}"'''.format(
         prop, cfg.get_value(path=DATA_CFG+'::'+prop)) for prop in dataset_props])
 
+    prop_string += f',TABLE_LOCATION:"{table_path}"'
+
     conn.query(f'''MERGE (n:dataset{{{prop_string}}})''')
 
     with CodeTimer(logger, 'setup proxy table'):
         # Reading dicom and opdata
+        logger.info("Loading!")
         df_dcmdata = spark.read.format("delta").load(dicom_path)
 
-        tuple_to_add = df_dcmdata.select("metadata.PatientName", "metadata.SeriesInstanceUID")\
-            .groupBy("PatientName", "SeriesInstanceUID")\
+        tuple_to_add = df_dcmdata.select("metadata.PatientID", "metadata.AccessionNumber", "metadata.SeriesInstanceUID")\
+            .groupBy("PatientID", "AccessionNumber", "SeriesInstanceUID")\
             .count()\
             .toPandas()
 
     with CodeTimer(logger, 'synchronize graph'):
-        dataset_name = cfg.get_value(path=DATA_CFG+'::DATASET_NAME')
+        dataset_name = cfg.get_value(path='DATA_CFG::DATASET_NAME')
```

Comment:
should this be `cfg.get_value(path=DATA_CFG+'::DATASET_NAME')`?

Reply:
@arfathpasha and I talked about this, either or is valid since DATA_CFG is being treated as a string literal.  Consistency within a file is probably ideal though, so fixed this